content/chainguard/chainctl-usage/chainctl-images-history.md

```diff
@@ -0,0 +1,150 @@
+│ │ ├ [3]
+│ │ ├ [3.13]
+│ │ ├ [3.13.3]
+│ │ └ [latest]
+```
+
+
+This will continue until all images (like `python` above) are listed with all their variants (releases like `3.13.3`). Notice that the list is not necessarily in order of release.
+
+What if I am interested in knowing how many times a variant has been built and for which platforms, along with the time and digests for each? Read on.
+
+
+## Examine images history
+
+To examine the history use this command, shown with an optional `--parent=$ORGANIZATION` switch demonstrating how to designate your org without using the menu shown earlier:
+
+```shell
+chainctl images history $IMAGE:$TAG --parent=$ORGANIZATION
+```
+
+
+For example, let's find the history of one of the `python` image variants from our previous list, `3.12.7`. So we enter:
+
+```shell
+chainctl images history python:3.12.7 --parent=chainguard.edu
+```
+
+
+The returned list is longer than we'll show here, but here's an excerpt:
+
+```shell
+- time: 2024-11-29 08:11:03 UTC
+  digest: sha256:16b52893f316d9d7074b9c24c30f82eab1e94356461439d4be1a62fe229e6933
+  architectures:
+    amd64: sha256:755b79b43c1e76472cc467c4636bdb75bf4c2fcc551103087df0a4b0dc039164 (23.14 MB)
+    arm64: sha256:d7dfaf24f292f490279611afcec49289aa528dba531590bebae059c7d3139ed6 (22.13 MB)
+- time: 2024-11-25 23:45:42 UTC
+  digest: sha256:025bbe734f9bb3550ea845f028f28c76c6dff742527e2764f4ba40b3773ee4f8
+  architectures:
+    amd64: sha256:c7b70882c6fe9b563aa5d5bdfb1a960c65b8c73e0830fc809ceb977e7f778e98 (23.14 MB)
+    arm64: sha256:864f735794264120aabdc9eda9e1126140762e1c11eed96a89b0ab2056cc3662 (22.13 MB)
+- time: 2024-11-22 01:32:55 UTC
+  digest: sha256:2a936c40669150e1a92e4c80b27410786925b8619c0ecc1679ec0a0f6b707235
+  architectures:
+    amd64: sha256:8db3676319588dca04664a4a57c9fa464398fa2fc6874d5a1500273cabdbde04 (23.16 MB)
+    arm64: sha256:ec8b8ef474ad6be50649cd9403882569d2cd1e71ac65e3bc263fc78aaef7608e (22.13 MB)
+
+...
+
+- time: 2024-10-02 23:56:00 UTC
+  digest: sha256:c7cf9f46124502b9e8aadf26b4c58e0cdbd5a08b0b97b6d3a451b89563a308e8
+  architectures:
+    amd64: sha256:d4606173598e7103015b37dec4894771bf9cc221db6f7b102006eb81962c0696 (23.43 MB)
+    arm64: sha256:f76a0a2f49418b030f3a31bcd2c8bddb8bbef7b006006aa59b74282955ab671d (22.41 MB)
+- time: 2024-10-01 23:40:56 UTC
+  digest: sha256:0b0daf09eeb92741efe0eae51dbdeea5a66bed870e0e00895630de729b233b7f
+  architectures:
+    amd64: sha256:637af1b20e5f8cee7e538b07a1ae3934297769216a65acc454e34fac3dcd3828 (23.46 MB)
+    arm64: sha256:8dce068942fa4dd155c87b6b8a3e4b8e2482a5fdb5232cb9dd73c39e63003038 (22.41 MB)
+
+```
+
+The command returns a reverse-chronological history of when a specific tag was updated to point to a new manifest digest. If images are not multi-arch, only a single digest without architecture will be displayed.
+
+You aren't required to include the release version tag. If you omit it, you will be presented with a menu, like this:
```

Comment:
I would just maybe rephrase this bit slightly to be more descriptive instead of relying on "like this". something like "When the release version tag is not provided, the command will present you with a menu that lets you select which tag you'd like to obtain the history for. For example:" then add the command, followed by "This will present you with a menu similar to this:" -- sort of ack'ing that this can change as new versions are released.

Reply:
I like that. I'll edit this. TY!